admin/bin/dev-shell

```diff
@@ -23,7 +26,18 @@ function docker_image() {
 }
 
 function docker_run() {
-    docker run --rm -ti ${DOCKER_RUN_ARGUMENTS:-} securedrop-admin "$@"
+    if [ -z "${CIRCLECI-}" ]
```

Comment:
Had never seen that form of expansion! "When not performing substring expansion, using the forms documented below (e.g., :-), bash tests for a parameter that is unset or null. <em>Omitting the colon results in a test only for a parameter that is unset.</em>". Nice.